### DISCUSSION TITLE
Where is the version `10.0.0` tag of `remark-cli`?

### DISCUSSION THREAD

**Discussion by dev-yakuza** [remarkjs/remark#868] [Q&A] [answered]
I think the `remark-cli` version is `10.0.0`.

- https://github.com/remarkjs/remark/commit/e55186f7d56c529d95ec7205572f016c73151201
- https://www.npmjs.com/package/remark-cli

But I couldn't find the `10.0.0` tag.

![Screen Shot 2021-09-30 at 13 04 08](https://user-images.githubusercontent.com/42969906/135385607-b9845dfc-2b27-4f11-bd60-64f0e26fffae.jpg)

I don't know you decide not to use the tag, or you just forget it. 🤔

**Comment by wooorm** [comment, marked as answer]
Hmm. Not a very conscious decisions, but the CLI release went together with https://github.com/remarkjs/remark/releases/tag/14.0.0.
I maintain a couple of monorepos with 50+ packages. There I also did all of their major releases “in” the main package’s major release.
That’s a more clear example where creating tons of tags would be unmanageable, but I just did the same here.

What are you using tags for?

**Comment by dev-yakuza** [comment]
@wooorm 
I'm a developer of [sider](https://sider.review/), and we provide a service for linters.

When we update linters, we create `news` on the document page. And we add it with the official tool link like the below.

https://github.com/sider/sider-docs/pull/698/files#diff-fc203eb3475a0dd117496f7f576202a05265930ffec2e1922bf1bfa867be60eeR34

For this update, we realized the tag doesn't exist, so I'm just asking you about the reason that is the mistake or decision. (if it was because of the decision, we also think about the link rule)

I understand your hard situation and thanks for your works, so we'll consider using another link. Thanks for the quick answer.

Just one more question, do you have a plan to keep the tags or it depends on a situation like this time? (Just I want to make sure that I can use the tag link in the future or not.)

**Reply by wooorm** [reply]
I’d appreciate it if you could use the 14.0.0 release link for this `remark-cli@10.0.0`. I’ll keep this in mind, keep my ears open for other people with similar problems, and will likely use `remark-cli@...` tags in the future again!
